engine/src/main/java/org/terasology/rendering/dag/nodes/BackdropNode.java

```diff
@@ -86,10 +81,9 @@ public void initialise() {
         new WireframeTrigger(renderingDebugConfig, this);
 
         addDesiredStateChange(new SetViewportToSizeOf(WRITEONLY_GBUFFER, displayResolutionDependentFBOs));
```

Comment:
I planned on doing this either way after asking for your opinion, glad you mentioned it on your own! :D